extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/devconsole/Name.java

```diff
@@ -101,6 +101,11 @@ static String createSimple(AnnotationInstance annotation) {
 
     @Override
     public int compareTo(Name other) {
+        // Quarkus classes should be last
```

Comment:
Is this really needed? I don't see a benefit of this change. I mean why should e.g. the `io.smallrye.config.inject.ConfigProducer` go before `io.quarkus.vertx.http.runtime.CurrentVertxRequest`?

Reply:
It is needed because you can have application beans that are generated.

Reply:
But _generated_ application beans are handled in the `DevBeanInfo#compare()`, or?

Reply:
My point is that you can have beans that are both application beans and generated beans and some come from Quarkus (with no user action).
So by doing this the Quarkus ones are moved to the bottom

Reply:
Well, I don't think that ordering matters here because users are probably not interested in generated beans anyway (be it a quarkus class or not). 

In other words, this should be enough (and already quite confusing ;-):
1. application beans, non-generated, sorted by providerType name
2. application beans, generated, sorted by providerType name
3. non-app beans, non-generated, sorted by providerType name
4. non-app beans, generated, sorted by providerType name

Reply:
I did not add this for no reason. I added it because of what I saw in a sample application.

Without the name comparison, things are still not perfect

Reply:
Hm, I wonder what these "application generated beans" are. Did you use a quickstart as a sample app?

Reply:
`io.quarkus.resteasy.reactive.server.runtime.NotFoundExceptionMapper$GeneratedExceptionHandlerFor$NotFoundException$OfMethod$toResponse`, `io.quarkus.resteasy.reactive.server.runtime.exceptionmappers.AuthenticationFailedExceptionMapper$GeneratedExceptionHandlerFor$AuthenticationFailedException$OfMethod$handle` among a few others.

These are always included with RESTEasy Reactive

Reply:
Anyway, it's not a blocker so I'm going to approve this pr ;-)

Reply:
That's my point. Without this change, they are displayed before the generated beans that are a result of the user writing some code.